adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.m

```diff
@@ -63,22 +106,56 @@ + (GADVersionNumber)adapterVersion {
 
 - (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
                    completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding banner ad.
+    [GADMediationAdapterPangle setCOPPA:adConfiguration.childDirectedTreatment.integerValue];
+    _bannerRenderer = [[GADPangleRTBBannerRenderer alloc] init];
+    [_bannerRenderer renderBannerForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
 - (void)loadInterstitialForAdConfiguration:(GADMediationInterstitialAdConfiguration *)adConfiguration
                          completionHandler:(GADMediationInterstitialLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding interstitial ad.
-}
-
-- (void)loadNativeAdForAdConfiguration:(GADMediationNativeAdConfiguration *)adConfiguration
-                     completionHandler:(GADMediationNativeLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding native ad.
+    [GADMediationAdapterPangle setCOPPA:adConfiguration.childDirectedTreatment.integerValue];
+    _interstitialRenderer = [[GADPangleRTBInterstitialRenderer alloc] init];
+    [_interstitialRenderer renderInterstitialForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
 - (void)loadRewardedAdForAdConfiguration:(GADMediationRewardedAdConfiguration *)adConfiguration
                        completionHandler:(GADMediationRewardedLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding rewarded ad.
+    [GADMediationAdapterPangle setCOPPA:adConfiguration.childDirectedTreatment.integerValue];
```

Comment:
Update to: 
```
    [GADMediationAdapterPangle setCoppa:adConfiguration.childDirectedTreatment];
```

Reply:
I update this invoke way ：
`if (adConfiguration.childDirectedTreatment) { [GADMediationAdapterPangle setCOPPA:adConfiguration.childDirectedTreatment.integerValue]; }`